downstream/assemblies/eda/assembly-eda-user-guide-overview.adoc

```diff
@@ -1,12 +1,20 @@
 [id="eda-user-guide-overview"]
-= {EDANameShort} user guide overview
 
-The following procedures form the user configuration:
-
-. Set up a new project
-. Set up decision environment
-. Set up token
-. Set up a rulebook activation
+= Red Hat {EDAcontroller} user guide overview
 
+The following procedures form the user configuration:
```

Comment:
Do they have to be done in this order?

Reply:
Yes and no, This is the logical ordering for sure. But the the first three can be done in any order. Creating a Rulebook requires  the top three to be done prior to its creation.